climada_petals/hazard/tc_tracks_forecast.py

```diff
@@ -362,6 +366,53 @@ def read_one_bufr_tc(self, file, id_no=None):
             else:
                 LOGGER.debug('Dropping empty track %s, subset %d', name, i)
 
+    def write_hdf5(self, file_name, complevel=5):
+        """Write TC tracks in NetCDF4-compliant HDF5 format. This method
+        overrides the method of the base class.
+        Parameters
+        ----------
+        file_name: str or Path
+            Path to a new HDF5 file. If it exists already, the file is overwritten.
+        complevel : int
+            Specifies a compression level (0-9) for the zlib compression of the data.
+            A value of 0 or None disables compression. Default: 5
```

Comment:
it's consistent with the parent class. we could make it a class attribute of the parent class so that it's inherited, but it's probably not that important.

Reply:
I suggest we leave it as it is. If we want to link it to the default of the parent class we would also have to think about a smart way to handle the docstring, so that the default is stated there as well. I thought about something like that for a while, but found no easy and readable solution.